openbb_terminal/econometrics/econometrics_controller.py

```diff
@@ -322,8 +323,6 @@ def call_load(self, other_args: List[str]):
             dest="examples",
         )
 
-        if other_args and "-" not in other_args[0][0]:
```

Comment:
Why was this removed?

This is quite important for me to stay as it makes for a much better UX

Reply:
The rationale here is that usage of the `-f` flag is the same in all the menus of the terminal.

If in the econometrics you can load a dataset without specifying that it comes from a file then one can likely assume the same behaviour in other menus where we don't have such argument pre-pending